Build/run.sh

```diff
@@ -11,6 +11,7 @@ usage() {
 #OUTDIR is bind mopunted and will contain the compiled output from the container
 OUTDIR='output'
 CONTNAME="$(whoami)-rzv2l_vlp_v3.0.0"
+test -t 1 && USE_TTY="-it"
```

Comment:
Can you please explain this, what does this do?

Reply:
I noticed that GitHub actions are not using a TTY device when running workflows. This results in errors when docker is executed with the parameter `-it`.
So with this command, I investigate if TTY device is being used.

Reply:
Thank you!